features/storeHealthCertificate.feature

```diff
@@ -8,9 +8,9 @@ Feature: Store Health Certificate
       And I set json body to the file at ./features/fixtures/storeHealthCertificate/Bundle-DDCCDocument.json
     When I receive a response
     Then I expect response should have a status 201
-      And I expect response header location should be /DDCCDocument/TESTID
-      And I store response at /DDCCDocument/TESTID as Bundle-DDCCDocument
-
+      And I expect response header location should be /Bundle/TESTID
+      And I expect the results from the GET request sent to path /Bundle/TESTID should have a json body like the file at ./features/fixtures/storeHealthCertificate/StoreHealthCertificate-response.json
```

Comment:
same here, will have to pull the path from the response location rather than hardcoding it.